modules/tinymce/src/core/main/ts/Rtc.ts

```diff
@@ -38,7 +38,7 @@ interface RtcRuntimeApi {
   hasRedo: () => boolean;
   transact: (fn: () => void) => void;
   applyFormat: (format: string, vars: Record<string, string>) => void;
-  removeFormat: (format: string, vars: Record<string, string>) => void;
+  removeFormat: (format: string, vars: Record<string, string>, similar?: boolean) => void;
```

Comment:
@spocke @TheSpyder not sure how you want this argument passed to RTC, so for now I've just added it as an optional argument here. Let me know if you'd prefer it done differently.

Reply:
From my initial checks RTC already works in the one case that uses `similar`, remove colour from `ColorSwatch`, but I'm logging a task to check.

For now please don't pass it to RTC (the code would ignore it, but I want the types to match 100%) and we'll do a separate PR if it turns out to be necessary.

Reply:
Done 👍

Reply:
Thanks! I recommend leaving `_similar` in the argument list, as we did with `_node`, I suspect with strict types the number of function arguments will matter 🤔

Reply:
I'm pretty sure it doesn't matter for strict types, but I made the change to keep it consistent anyways 👍